contracts/ethereum/GreeterL1.sol

```diff
@@ -6,9 +6,9 @@ import { ICrossDomainMessenger } from
     "@eth-optimism/contracts/libraries/bridge/ICrossDomainMessenger.sol";
 
 contract GreeterL1 {
-    address crossDomainMessengerAddr = 0x4361d0F75A0186C05f971c566dC6bEa5957483fD;
+    address crossDomainMessengerAddr = 0x5086d1eEF304eb5284A0f6720f79403b4e9bE294;
 
-    address greeterL2Addr = 0xD4c204223d6F1Dfad0b7a0b05BB0bCaB6665e0c9;
+    address greeterL2Addr = 0xC0836cCc8FBa87637e782Dde6e6572aD624fb984;
```

Comment:
This uses the same Greeter contract as the Optimism bridge tutorial, so I updated the addresses to reflect the new Goerli addresses, which can be found [here](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/cross-dom-comm/hardhat/contracts)